src/containers/Assets/index.tsx

```diff
@@ -20,12 +21,22 @@ const assets: Asset[] = [
     imageUrl: TnbLogoAndWordmark,
     subtext: 'TNB-Logo-and-Wordmark.Zip',
   },
+  {
+    downloadLink: `${socialMediaUrls.github}/Marketing/raw/master/Logo-and-Wordmark-white.zip`,
+    imageUrl: TnbLogoAndWordmarkWhite,
+    subtext: 'TNB-Logo-and-Wordmark-white.Zip',
```

Comment:
Just a suggestion: should we add a new property `color: light` or `color: dark` to each asset? this is to account for future assets that may also have light colors. Then the classname of the div should be dynamic like:

```
className={clsx('Assets__card-wrapper', {
          'Assets__card-wrapper--white': asset.color === 'light',
        })}
```

Not too sure about this tho. Let's see what the leads think about this.